recipe/patches/0006-allow-openssl-3.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/cmake/ssl.cmake b/cmake/ssl.cmake
```

Comment:
Would be nice to have some proper patch metadata and a commit message that explains the changes; basically do a `git init` for the unzipped tarball & commit the initial state (if there's no git repo to check out), then add the existing patches on top (if you remove your metadata-less patches first, you could do `find ../path/to/feedstock/recipe/patches/ | xargs git am`), and then finally do:
```
git format-patch <sha_of_baseline> --no-signature
```

Reply:
Thanks for reviewing! I've never encountered any directions for best practices when it comes to recipe patching, so this is a helpful suggestion. 👍